core/src/avm2/filters.rs

```diff
@@ -105,6 +116,9 @@ impl FilterAvm2Ext for Filter {
                 let gradientglowfilter = activation.avm2().classes().gradientglowfilter;
                 gradient_filter_to_avm2(activation, filter, gradientglowfilter)
             }
+            Filter::ShaderFilter(_) => Err(Error::RustError(
+                "Shader filter conversion not yet implemented".into(),
```

Comment:
Is it possible to implement this? I don't know how the impact would be, but anything that has a shader filter will now error whenever you access the filter list